lib/features/app_changelog/show_changelog.dart

```diff
@@ -0,0 +1,47 @@
+import "package:fast_immutable_collections/fast_immutable_collections.dart";
+import "package:flutter/material.dart";
+import "package:flutter_riverpod/flutter_riverpod.dart";
+import "package:package_info_plus/package_info_plus.dart";
+
+import "app_changelog.dart";
+import "repository/changelog_repository.dart";
+import "repository/local_changelog_repository.dart";
+
+Future<void> showChangelog(BuildContext context, WidgetRef ref) async {
+  final changelogs = await ref.watch(changelogRepositoryProvider.future);
```

Comment:
not sure if calling `ref.watch` outside `build` method of either a `Widget` or other riverpod's provider is safe. `ref.read` should be more appropiate in this case.